metricflow-semantics/tests_metricflow_semantics/model/semantics/test_linkable_spec_resolver.py

```diff
@@ -165,7 +165,7 @@ def test_create_linkable_element_set_from_join_path_multi_hop(  # noqa: D103
                 left_semantic_model_reference=SemanticModelReference("views_source"),
                 path_elements=(
                     SemanticModelJoinPathElement(
-                        semantic_model_reference=SemanticModelReference("bookings"),
+                        semantic_model_reference=SemanticModelReference("bookings_source"),
```

Comment:
I had to change this to `bookings_source` since `bookings` doesn't exist in `simple_manifest`, which was causing an assertion to break.